src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs

```diff
@@ -5329,6 +5330,189 @@ public async Task GracefulCloseDisablesReconnect()
         }
     }
 
+    [Fact]
+    public async Task HubMethodInvokesCreateActivities()
```

Comment:
An extra thing to test is verifying that spans created inside a hub invocation have the invocation span as its parent. For example, a hub invocation that then talks to the DB or sends a HTTP request has its spans correctly nested.